### ISSUE TITLE
This is awesome, thank you!

### ISSUE THREAD

**Issue by evelant** [codeque-co/codeque#1]
Thanks for making codeque, it's a fantastic tool! Are you planning to add replace functionality? Structural replace could do things that regex could never do.

**Comment by jayu** [comment]
Thank you! 

Yeah, it's planned, but it will take some time until I will get there. 

Here is the [roadmap](https://codeque.co/roadmap)

I plan to work on ESLint plugin first :D

**Comment by John0x** [comment]
@jayu is this currently being worked on?
Looks like it on the roadmap 👀

**Comment by jayu** [comment]
Yes, I have a working solution on branch, but it's not integrated with vscode extension yet.

I'm also considering to release replace and couple other features as a paid product. 

I've invested a lot of work into this project and would be cool to get something in return :D 

Roadmap is a bit outdated, I'm finishing support for python at the moment. I will update roadmap soon.

@John0x What's your use case for replace functionality?

**Comment by John0x** [comment]
Yeah, I get that. I'd maybe think about an enterprise plan or something?
I probably wouldn't pay for a structural replace feature, to be entirely honest, but thats just me :)

My use case are primarily large codebase refactorings. Something like https://comby.dev/ with proper IDE integration.

**Comment by jayu** [comment]
Thank you for your honest opinion!

With replace I aim to provide much more robust and flexible solution than comby offers, especially for nested code structures. 

I can't wait to share it, but it needs some polishing and improvements atm.
